app/adapters/tsheets_adapter.rb

```diff
@@ -0,0 +1,11 @@
+class TsheetsAdapter
+  def initialize
+    @tsheets_api = TSheets::API.new do |config|
+      config.access_token = ENV['TSHEETS_ACCESS_KEY']
```

Comment:
I generally propose to all the teams I work with to:
- avoid using ENV anywhere outside on initialization files (now then i think about it, those benefits are debatable, but experience showed that those helped avoid some issues)
-  use `ENV.fetch` at all times, in this case -- we will know that ENV variable is missing after we will launch rails app (it will throw an error), otherwise functionality will not be working without proper setup and we will notice it only by error logs (and waste more time on it).